src/chrome/content/rules/ImmobilienScout24.xml

```diff
@@ -0,0 +1,35 @@
+<ruleset name="ImmobilienScout24">
+
+	<target host="immobilienscout24.de" />
+	<target host="www.immobilienscout24.de" />
+	<target host="api.immobilienscout24.de" />
+	<target host="bonitaetscheck.immobilienscout24.de" />
+	<target host="ferien.immobilienscout24.de" />
+	<target host="floorplan.immobilienscout24.de" />
+	<target host="forward.immobilienscout24.de" />
+	<target host="home.immobilienscout24.de" />
+	<target host="kontakt.immobilienscout24.de" />
+	<target host="mietzahlungscheck.immobilienscout24.de" />
+	<target host="mobil.immobilienscout24.de" />
+	<target host="nachbarschaft.immobilienscout24.de" />
+	<target host="news.immobilienscout24.de" />
+	<target host="picture.immobilienscout24.de" />
+	<target host="regioad.immobilienscout24.de" />
+	<target host="service.immobilienscout24.de" />
+	<target host="sicherheit.immobilienscout24.de" />
+	<target host="tracking.immobilienscout24.de" />
+	
+	<!-- not working:
+		blog.immobilienscout24.de
+		developer.immobilienscout24.de
+		forum.immobilienscout24.de
+		www.focus.immobilienscout24.de
+	-->
+
+	<rule from="^http://(?:www\.)?immobilienscout24\.de/"
```

Comment:
Please remove the `?:` and also add a comment for the mismatch on ^immobilienscout24.de
Comments should be on the top above the ruleset.